src/lib.rs

```diff
@@ -56,7 +56,7 @@ use core::convert::{TryFrom, TryInto};
 use bitcoin::address::NetworkValidation;
 
 pub use de::{DeserializeParams, DeserializationState, DeserializationError};
-pub use ser::{SerializeParams};
+pub use ser::SerializeParams;
```

Comment:
`cargo clippy` fix

Reply:
I'm glad you pointed this out in a comment @realeinherjar. I like to see formatting changes in their own commit.

Reply:
Done!